apps/meteor/app/cloud/server/functions/syncWorkspace/syncCloudData.ts

```diff
@@ -74,14 +75,41 @@ export async function syncCloudData() {
 
 		await callbacks.run('workspaceLicenseChanged', license);
 
+		SystemLogger.info({
+			msg: 'Synced with Rocket.Chat Cloud',
+			function: 'syncCloudData',
+		});
+
 		return true;
 	} catch (err) {
+		/**
+		 * If some of CloudWorkspaceAccessError and CloudWorkspaceRegistrationError happens, makes no sense to run the legacySyncWorkspace
+		 * because it will fail too.
+		 * The DuplicatedLicenseError license error is also ignored because it is not a problem. the Cloud is allowed to send the same license twice.
+		 */
+		switch (true) {
+			case err instanceof CloudWorkspaceAccessError:
+			case err instanceof CloudWorkspaceRegistrationError:
+			case err instanceof DuplicatedLicenseError:
+				SystemLogger.info({
```

Comment:
```suggestion
				SystemLogger.error({
```

Reply:
Considering the comment above, those errors are suppressed since everything is under control, so I decide to handle them as info (if the admin wants to follow all the steps), but since we handled its not an exception anymore